o1vm/src/interpreters/riscv32i/mod.rs

```diff
@@ -1,12 +1,12 @@
 /// The minimal number of columns required for the VM
 // FIXME: the value will be updated when the interpreter is fully
 // implemented. Using a small value for now.
-pub const SCRATCH_SIZE: usize = 20;
+pub const SCRATCH_SIZE: usize = 80;
```

Comment:
I guess we can try to set a value the no-action program requires atm?

Reply:
i just chose this value because it worked during testing, a size of 20 was too small.